civihr_employee_portal/civihr_employee_portal.info

```diff
@@ -45,6 +45,7 @@ files[] = views/includes/civihr_employee_portal_argument_age_group.inc
 files[] = views/includes/civihr_employee_portal_handler_absence_duration.inc
 files[] = views/includes/civihr_employee_portal_handler_absence_entitlement.inc
 files[] = views/includes/civihr_employee_portal_handler_activity_type.inc
+files[] = views/includes/civihr_employee_portal_handler_absence_table_header_block_area.inc
```

Comment:
maybe I Am missing some stuff here but shouldn't you deleted any reference to the views you replaced in this PR as well any related files.